docs/cep/tutorials/transforming-data.md

```diff
@@ -22,70 +22,61 @@ the temperature of a room every second are transformed to indicate the average t
 5. Add the following sample stream application.
```

Comment:
In all cases, change "stream application" to "stream worker."

They used to call stream workers "stream applications." This is now outdated terminology, like calling Macrometa C8DB.